elyra/pipeline/processor_kfp.py

```diff
@@ -77,23 +77,37 @@ def process(self, pipeline):
 
         pipeline_name = pipeline.name
         try:
-            # Connect to the KFP server and determine whether
-            # a pipeline with this name was already uploaded
-            session_cookie = None
-
-            if api_username and api_password:
-                endpoint = api_endpoint.replace('/pipeline', '')
-                session_cookie = self._get_user_auth_session_cookie(endpoint,
-                                                                    api_username,
-                                                                    api_password)
+            # Connect to the Kubeflow server, determine whether it is secured,
+            # and if it is try to authenticate with the user-provided credentials
+            # (if any were defined in the runtime configuration)
+
+            endpoint = api_endpoint.replace('/pipeline', '')
+            auth_info = \
+                KfpPipelineProcessor._get_user_auth_session_cookie(endpoint,
+                                                                   api_username,
+                                                                   api_password)
+
+            self.log.info(f"Kubeflow endpoint '{api_endpoint}' secured: "
```

Comment:
This feels more like a debug statement to me.

Reply:
I changed the log level to debug and dumped the entire data structure.